bittensor/_dendrite/dendrite_impl.py

```diff
@@ -436,7 +436,80 @@ def text_causal_lm (
         self.update_stats( formatted_endpoints, synapses, formatted_inputs, outputs, codes, times )
         return outputs[0], codes[0], times[0]
 
-    def text_last_hidden_state( 
+    def text_causal_lm_next(
```

Comment:
Should we expect this synapse to be called on its own or always in conjunction with CausalLM?

Reply:
TextCausalLMNext can be called independently of TextCausalLM, but for initial tests they'll both run so comparisons can be performed.